src/pages/kb/data-sources/querying-urls.md

```diff
@@ -1,28 +1,56 @@
 ---
 category: querying
 parent_category: data-sources
-title: Querying URLs
+title: Querying URLs & JSON APIs
 slug: urls
+toc: true
 ---
-Using a URL based data source requires that the URL use our format (see example below).
 
-The query itself inside Redash will simply contain the URL to be executed
-(i.e.   <http://myserver/path/myquery>) and result with all the data it has.
+Sometimes you need to visualize data not contained in an RDBMS or NOSQL data store. For those times, Redash provides two URL-based mechanisms to pull in your data: `URL` and `JSON`. Either source can be added from the settings screen in Redash.
 
-Here's a list of valid column types returned in results:
+`URL` is powerful because you can load large amounts of tabular data while abstracting away the underlying storage mechanism. This data source allows you to explicitly declare the types of data returned (text, dates, numbers e.g.). If a traditional database connection is unavailable, `URL` is the best way to load data to Redash.
```

Comment:
The goal is to deprecate the `URL` data source. There is no point in keeping it except for legacy reasons.

Reply:
I updated the body text and added a callout about upcoming deprecation.